features/drug_effect.feature

```diff
@@ -0,0 +1,25 @@
+        Scenario: Drug effect implementation
+
+Feature: Drug effect validation
+                Given I get target
+                When I should see the numerical target
+                And I should see target number should be in the range of citizens list
+                And I should see target is relevant to alive citizens
+                Then validation method should return True
+
+Feature: Drug effect activation
+                Given activate method is called
+                Then active player should get message "Выберите гражданина, к дому которого будет отправлен Наркоман..."
+                And active player should enter the target number
+                When validation is successful
+                Then target number should be set as effect target
+                When validation is failed
+                Then activation should be repeated
+
+Feature: Drug effect resolve
+                Given resolve method is called and effect status should be ACTIVATED
+                When camera, alarm, trap, or witness effect is in the target citizen effects list
+                Then clear relevant effect from the target citizen effects list
```

Comment:
All the effects are cleared, or just one?
(In case if, for example: alarm, trap, camera are active at the same time?)

Reply:
relevant effects shouldn't be removed. But rather marked as FINISHED.

Reply:
Yes, my bad, i'll fix it, thanks)